File: server/Model/Schema.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Server.Model
{
    public class SchemaDB
    {
        public List<SchemaTable> Tables { get; set; } = new List<SchemaTable>();
    }

    public class SchemaTable
    {
        public string Name { get; set; }

        public List<string> Fields { get; set; } = new List<string>();
    }
}

File: ClientApp/src/app/model/schema.ts
export class SchemaTable {
    public Name: string;
    public Fields: string[] = [];
}

export class SchemaDB {
    public Tables: SchemaTable[] = [];
}

export class EntityInfo {
    public TableName: string;
    public ClassName: string;
    public FriendlyName: string;
}

export class FieldInfo {
    public Name: string;
    public DisplayName: string;
    public FriendlyDisplayName: string;
}